From 5ca94ce9d04f3fb5872026fdf7d3b84a55bff255 Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Tue, 29 Nov 2011 18:18:16 +0530
Subject: [PATCH 0631/1092] arm:omap:am33xx: Create an alias for pruss clock

PRUSS driver has specific requriement for clock naming
convention, due to legacy platform support.
So create an alias for the clock "icss_uart_gclk" --> "pruss".

Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index f33f8c7..727d3fe 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1803,6 +1803,9 @@ static void __init am335x_evm_init(void)
 	usb_musb_init(&musb_board_data);
 	omap_board_config = am335x_evm_config;
 	omap_board_config_size = ARRAY_SIZE(am335x_evm_config);
+	/* Create an alias for icss clock */
+	if (clk_add_alias("pruss", NULL, "icss_uart_gclk", NULL))
+		pr_err("failed to create an alias: icss_uart_gclk --> pruss\n");
 }
 
 static void __init am335x_evm_map_io(void)
-- 
1.7.11.2

